Client/Frontend/Autofill/CreditCard/ViewComponents/RemoveCardButton.swift

```diff
@@ -62,6 +62,9 @@ struct RemoveCardButton: View {
         .onChange(of: themeVal) { newThemeValue in
             applyTheme(theme: newThemeValue.theme)
         }
+        .onReceive(NotificationCenter.default.publisher(for: UIApplication.willResignActiveNotification)) { _ in
+            showAlert = false
+        }
```

Comment:
You could try `@Environment (\.scenePhase) private var scenePhase` however using the publisher is fine as I have had reliability issues by using `scenePhase` environment variable. Maybe it's fixed now 🤷🏼

Reply:
@petruSt24, can you comment on why you are making `showAlart = false` as you need the alert to dismiss before dismissing the view?

A slight drawback with this approach is that even after a user authenticates, you still don't see the alert. Confirm with UX before merging this please

Reply:
there's a mechanism in place that handles the dismissal of screens when authentication fails. That mechanism will fail when an alert controller is also present in that screen and the RemoveCardButton doesn't nor should it have access to the user's authentication status for security reasons

Reply:
@petruSt24 the comment goes inside the onReceive not on github 😅 
Summarize it in a sentence so folks are aware when they look at the code please